mcc/ttl/main.go

```diff
@@ -132,11 +133,13 @@ func UpsertResourceRecordSetTTL(
 		log.Panic(err.Error())
 	}
 	// Submit batch changes
-	changeResponse, err = svc.ChangeResourceRecordSets(changeRRSInput)
+	if !dryrun {
+		changeResponse, err = svc.ChangeResourceRecordSets(changeRRSInput)
```

Comment:
Shouldn't be `changeResponse` getting some content back to return from this function?